xgi/generators/nonuniform.py

```diff
@@ -418,15 +424,24 @@ def random_flag_complex(N, p, seed=None, max_order=2):
         Probabilities (between 0 and 1) to create an edge
         between any 2 nodes
 
+    max_order : int
+        maximal dimension of simplices to add to the output simplicial complex
+
     Returns
     -------
-    hyperedges_final : list of tuples
-        List of hyperedges, i.e. tuples of length 2 and 3.
+    S : xgi.SimplicialComplex
 
     Notes
     -----
     Computing all cliques quickly becomes heavy for large networks.
 
+    Example
+    -------
+    >>> import xgi
+    >>> N = 100
+    >>> p = 0.05
+    >>> S = xgi.random_flag_complex(N, p, max_order=2)
+
```

Comment:
I'd get rid of this example for two reasons: 1) it doesn't actually illustrate any functionality, it just shows a function call, and 2) it slows down the doctest run.

Reply:
Done.